src/awkward/_v2/forms/unionform.py

```diff
@@ -234,6 +234,10 @@ def fields(self):
         fieldslists = [cont.fields for cont in self._contents]
         return list(set.intersection(*[set(x) for x in fieldslists]))
 
+    @property
+    def is_tuple(self):
+        return all(x.is_tuple for x in self._contents)
```

Comment:
This is a good definition, but the v1 logic doesn't match it. (Unless I'm just unable to think straight on short notice. Anyway, this is a prompt to check it.)